Error occured in ajaxTWUserMentions:
Traceback (most recent call last):
  File "F:\Libraries\Documents\ASPIRA\Social-Network-Harvester-v1.0\SocialNetworkHarvester_v1p0\Twitter\views.py", line 123, in ajaxTWFollowersTable
    queryset = Follower.objects.filter(twuser=twUser)
NameError: name 'Follower' is not defined
Error occured in ajaxTWUserMentions:
Traceback (most recent call last):
  File "F:\Libraries\Documents\ASPIRA\Social-Network-Harvester-v1.0\SocialNetworkHarvester_v1p0\Twitter\views.py", line 123, in ajaxTWFollowersTable
    queryset = follower.objects.filter(twuser=twUser)
NameError: name 'follower' is not defined
